app/src/main/java/com/duckduckgo/app/browser/omnibar/OmnibarLayout.kt

```diff
@@ -144,12 +164,21 @@ open class OmnibarLayout @JvmOverloads constructor(
     @Inject
     lateinit var pixel: Pixel
 
+    @Inject
+    lateinit var duckChat: DuckChat
+
     @Inject
     lateinit var dispatchers: DispatcherProvider
 
     @Inject
     lateinit var appPersonalityFeature: AppPersonalityFeature
 
+    @Inject
+    lateinit var visualDesignExperimentDataStore: VisualDesignExperimentDataStore
+
+    private var isInitialRender = true
```

Comment:
can we rename this to something related to the animation?

Reply:
Removed the boolean, using the null state